libgloss/arc/nsim-syscalls.c

```diff
@@ -206,7 +206,7 @@ _fstat (int fd, struct stat *buf)
 {
   struct nsim_stat nsim_stat;
   long __res;
-  _naked_syscall2 (__res, stat, fd, &nsim_stat)
+  _naked_syscall2 (__res, fstat, fd, &nsim_stat)
```

Comment:
Is this change compatible with nSIM simulator, also you need to change the same file in arc64.